Sources/NIOHTTP2/HTTP2Stream.swift

```diff
@@ -108,6 +110,14 @@ final class HTTP2Stream {
     /// Whether this stream is still active on the connection. Streams that are not active on the connection are
     /// safe to prune.
     var active: Bool = false
+    
+    /// An HPACK decoder for this stream, handling integer and Huffman decoding and the management of a dynamic
+    /// header table.
+    var decoder: HPACKDecoder
+    
+    /// An HPACK encoder, handling integer and huffman encoding, header packing, and dynamic header table
+    /// management.
+    var encoder: HPACKEncoder
```

Comment:
No need to add these here now, it's just a bunch of memory allocation that the user doesn't need yet.

Reply:
Ok.